index.html

```diff
@@ -495,104 +495,107 @@ <h2>Controller Documents</h2>
               </tr>
             </thead>
             <tbody>
               <tr>
                 <td>`id`</td>
                 <td>yes</td>
                 <td>
-A <a data-cite="INFRA#string">string</a> that conforms to the URL syntax
-defined in Section [[[#subjects]]].
+A <a data-cite="INFRA#string">string</a> that conforms to the URL syntax.
                 </td>
+                <td style="white-space: nowrap;">[[[#subjects]]]</td>
               </tr>
               <tr>
-                <td>`alsoKnownAs`</td>
+                <td>`controller`</td>
                 <td>no</td>
                 <td>
-A <a data-cite="INFRA#ordered-set">set</a> of
-<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL syntax
-defined in the [[[URL]]] as defined in Section [[[#also-known-as]]].
+A <a data-cite="INFRA#string">string</a> or a
+<a data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL
+syntax.
                 </td>
+                <td style="white-space: nowrap;">[[[#controllers]]]</td>
               </tr>
               <tr>
-                <td>`controller`</td>
+                <td>`alsoKnownAs`</td>
                 <td>no</td>
                 <td>
-A <a data-cite="INFRA#string">string</a> or a
-<a data-cite="INFRA#ordered-set">set</a> of
-<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL syntax
-defined in the [[[URL]]] as defined in Section [[[#controllers]]].
+A <a data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL
+syntax.
                 </td>
+                <td style="white-space: nowrap;">[[[#also-known-as]]]</td>
               </tr>
               <tr>
                 <td>`verificationMethod`</td>
                 <td>no</td>
                 <td>
 A <a data-cite="INFRA#ordered-set">set</a> of
-[=verification method=] <a data-cite="INFRA#ordered-map">maps</a>, each of which
-conforms to the rules in Section [[[#verification-methods]]].
+[=verification method=] <a data-cite="INFRA#ordered-map">maps</a>.
                 </td>
+                <td style="white-space: nowrap;">[[[#verification-methods]]]</td>
               </tr>
               <tr>
                 <td>`authentication`</td>
                 <td>no</td>
                 <td>
-A <a data-cite="INFRA#ordered-set">set</a> of
-[=verification method=] <a data-cite="INFRA#ordered-map">maps</a>, each of which
-conforms to the rules in Section [[[#authentication]]]; or a
-a <a data-cite="INFRA#ordered-set">set</a> of
-<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL syntax
-defined in the [[[URL]]].
+A <a data-cite="INFRA#ordered-set">set</a> of [=verification method=] <a
+data-cite="INFRA#ordered-map">maps</a> or a a <a
+data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL
+syntax.
```

Comment:
This could be read as requiring the maps to conform to the URL syntax. Therefore —
```suggestion
A <a data-cite="INFRA#ordered-set">set</a> of <a
data-cite="INFRA#string"> strings</a>, each of which conforms to
the URL syntax, or a <a data-cite="INFRA#ordered-set">set</a> of
[=verification method=] <a data-cite="INFRA#ordered-map">maps</a>.
```